src/stub-request.js

```diff
@@ -42,7 +42,7 @@ function stubRequest (type, definition=identity) {
       [LP_API]: {
         isStub: true,
         stubData: createActionOptions(definition, args),
-        requestKey: type,
+        type,
```

Comment:
What are your thoughts on renaming this utility to `createStubRequest`? It seems like a closer match to the API

Reply:
👍 this is the time to do it.